src/org/traccar/ServerManager.java

```diff
@@ -88,7 +88,7 @@ public void start() throws Exception {
             try {
                 server.start();
             } catch (BindException e) {
-                LOGGER.warn("One of the protocols is disabled due to port conflict");
+                LOGGER.warn("Port " + server.getPort() + " is disabled due to conflict");
```

Comment:
Please use formatting version of the method instead of concatenating it yourself.